docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,366 @@
+
+#### Do we allow circular references between registries?
+- Short answer: No.
+- Long answer: Circular references are already prohibited inside a registry, so circular references between
+  registries can lead to the kind of problems (e.g. unresolvable references). Weaver will detect and report
+  an error if a circular reference is detected between registries.
+
+#### Do we allow references to groups or attributes defined in an imported registry?
+- Short answer: Yes.
+- Long answer: References to groups or attributes defined in an imported registry are allowed. This feature
+  allows a registry to extend or override the attributes of an imported group. Weaver will detect and report an
+  error if a reference is made to an entity that is not defined in the imported registry.
+
+### Open Questions:
+- Can we override any field of a group defined in an imported registry? No for `type` field, what about the other
+  fields?
+- Is there a relationship to define between the instrumentation scope name and version and the registry?
+
+
+## Core Policies
+
+These core policies aim to ensure consistency and backward compatibility across registries.
+
+### Group Policies
+- No group ID collisions
+- Only attribute groups are allowed in the attribute registry
+- Ref attributes are not allowed in the attribute registry
+- ID attributes are not allowed in groups outside of the attribute registry
+
+### Attribute Policies
+- No attribute name collisions
+- Attributes cannot be removed
+- Attributes cannot "degrade" in stability (e.g., stable -> experimental)
+- Stable attributes cannot change type
+- All attributes must be defined inside a semantic convention registry
+- Attribute names must match the following regular expression: `^[a-z][a-z0-9]*([._][a-z0-9]+)*$`
+- Attributes with prefixes are not allowed; attributes must be fully qualified
+
+### Enum Member Policies
+- Stable members cannot change stability
+- Values cannot change
+- IDs cannot be removed
+- Enum IDs must match the following regular expression: `^[a-z][a-z0-9]*([._][a-z0-9]+)*$`
+
+### Metrics
+- No metric name collisions
+- Metrics cannot be removed
+- Metrics cannot "degrade" in stability (e.g., stable -> experimental)
+- Stable metric units cannot change
+- Stable metric instruments cannot change
+- The set of required/recommended attributes must remain the same
+- Metric names must match the following regular expression: `^[a-z][a-z0-9]*([._][a-z0-9]+)*$`
+
+### Events
+- No event name collisions
+- Events cannot be removed
+- Events cannot "degrade" in stability (e.g., stable -> experimental)
+- The set of required/recommended attributes must remain the same
+- Event names must match the following regular expression: `^[a-z][a-z0-9]*([._][a-z0-9]+)*$`
+
+### Spans
+- No span name collisions
+- Spans cannot be removed
+- Spans cannot "degrade" in stability (e.g., stable -> experimental)
```

Comment:
I think this should be reversed.

E.g. I'm adding semconv for Azure messaging library based on stable generic semconv (they are not stable yet, but let's assume they are).

My customizations to generic messaging spans and metrics are experimental, core conventions are stable. Why is it not allowed?

However I should not be able to define stable conventions based on generic experimental ones - this would lead to breaking changes.

Reply:
I agree, that should be allowed, and I will make the change.

Reply:
Reference experimental entities is now allowed by default.